docs/source/conf.py

```diff
@@ -39,6 +39,7 @@
     "sphinx_autodoc_typehints",
     "nbsphinx",
     "sphinx_gallery.load_style",
+    "IPython.sphinxext.ipython_console_highlighting",
```

Comment:
for what purpose? @pseusys ?

Reply:
The docs wouldn't build without it

Reply:
As far as I know this extension shouldn't be needed for building any of the existing examples. Moreover, I was able to build documentation without this line. Maybe you could specify the error message that this line prevents?